<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KanbanButtonsView</class>
 <widget class="QWidget" name="KanbanButtonsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="mButtonKanbanCreate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolTip">
      <string>New Kanban [Ctr+K, Ctrl+N]</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>New Kanban</string>
     </property>
     <property name="icon">
      <iconset resource="../../Kanban/kanban.qrc">
       <normaloff>:/images/Light/button/add.png</normaloff>:/images/Light/button/add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+K, Ctrl+N</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="mButtonKanbanRename">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Rename Kanban [Ctr+K, Ctrl+R]</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Edit Kanban</string>
     </property>
     <property name="icon">
      <iconset resource="../../Kanban/kanban.qrc">
       <normaloff>:/images/Light/button/rename.png</normaloff>:/images/Light/button/rename.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+K, Ctrl+R</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="mButtonKanbanDelete">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Delete Kanaban [Ctr+K, Ctrl+D]</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Delete Kanban</string>
     </property>
     <property name="icon">
      <iconset resource="../../Kanban/kanban.qrc">
       <normaloff>:/images/Light/button/delete.png</normaloff>:/images/Light/button/delete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+K, Ctrl+D</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="mButtonColumnCreate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolTip">
      <string>New Column [Ctr+C, Ctrl+N]</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>New Column</string>
     </property>
     <property name="icon">
      <iconset resource="../../Kanban/kanban.qrc">
       <normaloff>:/images/Light/button/add.png</normaloff>:/images/Light/button/add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+C, Ctrl+N</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="mButtonColumnRename">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Rename Column [Ctr+C, Ctrl+R]</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Edit Column</string>
     </property>
     <property name="icon">
      <iconset resource="../../Kanban/kanban.qrc">
       <normaloff>:/images/Light/button/rename.png</normaloff>:/images/Light/button/rename.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+C, Ctrl+R</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="mButtonColumnDelete">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Delete Column [Ctr+C, Ctrl+D]</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Delete Column</string>
     </property>
     <property name="icon">
      <iconset resource="../../Kanban/kanban.qrc">
       <normaloff>:/images/Light/button/delete.png</normaloff>:/images/Light/button/delete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+C, Ctrl+D</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Kanban/kanban.qrc"/>
 </resources>
 <connections/>
</ui>
